books/free-programming-books-id.md

```diff
@@ -29,6 +30,11 @@
 * [Tutorial Membuat Aplikasi Galeri Foto Android](https://www.smashwords.com/books/view/533096)
 
 
+### C
+
+* [Belajar C dari dasar](https://www.petanikode.com/c-untuk-pemula/) - Ahmad Muhardian
```

Comment:
```suggestion
* [Belajar Pemrograman C untuk Pemula](https://www.petanikode.com/tutorial/c/) - Ahmad Muhardian (:construction: *in process*)
```

I've included a more generic link that contains the list of guides and I've modified a few things

Reply:
It seems article was updated recently. Maybe changes title since bumped PR was posted :wink: